src/views/components/admin/graphicalParkingLot.js

```diff
@@ -7,7 +7,7 @@ function GraphicalParkingLot({ ParkingSpaces }) {
         const currentTime = new Date(); // Now
 
         // Check if the space is reserved or blocked and should remain so regardless of time
-        if (parkingSpace.Status === 'blocked' || parkingSpace.Status === 'reserved') {
+        if (parkingSpace.Status === 'blocked' || parkingSpace.Status === 'reserved' || parkingSpace.Status === 'occupied') {
```

Comment:
Hm, this shouldn't be there, because it only applies to blocked and reserved status...